surveys/serializer_field_generators.py

```diff
@@ -4,6 +4,23 @@
 from .generators import BaseFieldGenerator as BaseBaseFieldGenerator
 
 
+class MultipleIntegerSerializer(serializers.Serializer):
+    """Dynamically generate a serializer with IntegerField based on the choices"""
```

Comment:
I didn't call it `Field` because it does not extend `Field` and although `Fields` are serializers they add some extra functionality.